ratelimit/redis_test.go

```diff
@@ -87,6 +89,37 @@ func Test_clusterLimitRedis_WithPass(t *testing.T) {
 	}
 }
 
+func Benchmark_clusterLimitRedis_Allow(b *testing.B) {
+	redisAddr, done := redistest.NewTestRedis(b)
+	defer done()
+
+	for i := 0; i < 21; i++ {
+		b.Run(fmt.Sprintf("ratelimit with group name of 2^%d symbols", i), func(b *testing.B) {
```

Comment:
minor: I think it is cleaner to format final decimal result (i.e. 1<<i) into benchmark name